docs/tutorial/getting-started.md

```diff
@@ -0,0 +1,43 @@
+---
+id: getting-started
+title: Getting started with Tartiflette
+sidebar_label: 1. Getting started
+---
+
+Welcome, either you are a GraphQL newbie or an advanced user, Tartiflette will help you to build a powerful GraphQL API.
+
+This tutorial will explain to you the main features of Tartiflette and some of its advanced features _(dynamic introspection, SDL Directives ...)_ 
+
+## What're we doing?
+
+Nothing is better than to code to understand a library. Let's go to develop your **Tartiflette recipes manager GraphQL API**.
```

Comment:
Nothing is better than coding to understand a library. Let's develop your own **Tartiflette recipes manager in a GraphQL API**.

Reply:
👍